docs/content/latest/integrations/jdbc-driver.md

```diff
@@ -0,0 +1,276 @@
+
+{{< note >}}
+Note: The driver requires YugabyteDB version 2.7.2.0 or higher, and Java 8 or above.
+{{< /note>}}
+
+### Install YugabyteDB and create a local Cluster
+
+We are using the [yb-ctl](https://docs.yugabyte.com/latest/admin/yb-ctl/#root) utility in this tutorial.
+
+Create a universe with a 3-node RF-3 cluster with some fictitious geo-locations assigned. Please note the placement values here are just tokens and have nothing to do with actual aws cloud, regions and zones.
+
+```sh
+$ cd <path-to-yugabytedb-installation>
+
+./bin/yb-ctl create --rf 3 --placement_info "aws.us-west.us-west-2a,aws.us-west.us-west-2a,aws.us-west.us-west-2b"
+```
+
+### Check Uniform load balancing using yb-sample-apps
+
+- Download yb-sample-apps JAR file
+
+  ```sh
+  wget https://github.com/yugabyte/yb-sample-apps/releases/download/v1.4.0/yb-sample-apps.jar 
+  ```
+
+- Now, run the SqlInserts workload application which creates multiple threads that perform read and write operations on a sample table created by the app. Uniform load balancing is enabled by default in all Sql* workloads of the yb-sample-apps including SqlInserts.
+
+  ```sh
+  java -jar yb-sample-apps.jar  \
+       --workload SqlInserts \
+       --num_threads_read 15 --num_threads_write 15 \
+       --nodes 127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433
+  ```
+
+- The above application will create 30 connections, 1 for each reader and writer threads. To verify the behavior, wait for the app to create connections and then visit `http://<host>:13000/rpcz` from your browser for each node to see that the connections are equally distributed among the nodes. 
+The above URL presents a list of connections where each element of the list has some information about the connection as shown in the following screenshot. You can count the number of connections from that list, or simply search for the occurrence count of the ‘host’ keyword on that webpage. Each node should have 10 connections.
+  
+  <br/><br/>
+  ![Load balancing with host connections](/images/develop/ecosystem-integrations/jdbc-load-balancing.png)
+
+### Check Topology-aware load balancing using yb-sample-apps
+
+- For Topology Aware Load Balancing run the SqlInserts workload application with topology-keys option as `aws.us-west.us-west-2a`, so only two nodes will be used in this case.  
+
+  ```sh
+  java -jar yb-sample-apps.jar \
+        --workload SqlInserts \
+        --nodes 127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433 \
+        --num_threads_read 15 --num_threads_write 15 \
+        --topology_keys aws.us-west.us-west-2a
+  ```
+
+- To verify the behavior, wait for the app to create connections and follow the same steps as before. The first two nodes should have 15 connections each, and the third node should have zero connections.
```

Comment:
To verify the behavior, wait for the app to create connections and then navigate to `http://<host>:13000/rpcz`. The first two nodes should have 15 connections each, and the third node should have zero connections.